tensorzero-internal/tests/e2e/clickhouse.rs

```diff
@@ -102,17 +339,25 @@ async fn test_clickhouse_migration_manager() {
                 );
             }
 
-            let clean_start =
-                migration_manager::run_migration(migrations[migration_num].as_ref(), true)
+            let run_migration = || async {
```

Comment:
if we set all these to true then the test takes ~2min. That seems a lot longer than the insert would take since ClickHouse should be able to do that quickly. So I assume this is due to all the sleeping this flag does. Do you think you could refactor to make this faster?